include/aspect/material_model/interface.h

```diff
@@ -840,7 +840,7 @@ namespace aspect
          * 0, then the viscosity does not need to be computed.
          */
         virtual void evaluate(const MaterialModel::MaterialModelInputs<dim> &in,
-                              MaterialModel::MaterialModelOutputs<dim> &out) const = 0;
+                              MaterialModel::MaterialModelOutputs<dim> &out) const;
```

Comment:
Why do we now provide a default implementation for evaluate? Should a material model not be forced to implement this function?